code/backend/testscript.py

```diff
@@ -0,0 +1,70 @@
+#from flask_cors import CORS
+#from flask import Flask, request, jsonify# Adjust according to your structure
+
+#app = Flask(__name__)
+#CORS(app)
+
+#@app.route('/api/getData', methods = ['POST'])
+
```

Comment:
Try something like this 

@app.route('/login', methods=['POST', 'GET'])
def login():
    error = None
    if request.method == 'POST':
        if valid_login(request.form['username'],
                       request.form['password']):
            return log_the_user_in(request.form['username'])
        else:
            error = 'Invalid username/password'
    # the code below is executed if the request method
    # was GET or the credentials were invalid
    return render_template('login.html', error=error)

Reply:
fixed the test script